integrationtests/tests/tests.rs

```diff
@@ -142,7 +150,7 @@ async fn peg_outs_must_wait_for_available_utxos() {
 
 #[tokio::test(flavor = "multi_thread")]
 async fn ecash_can_be_exchanged_directly_between_users() {
-    let (fed, user_send, bitcoin, _, _) = fixtures(4, &[sats(100), sats(1000)]).await;
+    let (fed, user_send, bitcoin) = Fixtures::new(4, &[sats(100), sats(1000)]).build().await;
```

Comment:
Unless some extra optionally is planed in the future, that split into `new()` and `build()` seems unnecessary. `Fixtures::make_all(...)` and `::make_smth(...)` would be better, IMO.

Reply:
Ok yes I'll chagne the naming. However having `new` and then the `build` was necessary I tried without it by calling the `new``setup` and the `builds` `news` but they have to share a state if `new_all` wanted to call `new`, the other option would have been to repeat everything